source/architecture.rst

```diff
@@ -1,11 +1,89 @@
-Architecture of S-ENDA
-"""""""""""""""""""""""
+Architecture of S-ENDA Find
+"""""""""""""""""""""""""""
 
-..
+S-ENDA architecture is described using the C4 model (https://c4model.com/)
+C4 does not define any properties based on the directionality
+of the used arrows so each arrow should have a textual
+description to avoid disambiguity.
 
-   S-ENDA architecture is described using the C4 model (https://c4model.com/)
+We draft the initial version of the architecture with emphasis
+on the goal and use cases of the first sprint of the project.
+
+The goal
+========
+To draft the architecture for S-ENDA find and access,
+and establish metadata handling to lift the user experience
+in finding dynamic geodata to a new level
+
+
+The use cases
+=============
+- An outdoor swimming competition organizer
+  wants to know sea-water temperature at the event location #37
+- A user shall extract observed and forecasted temperature
+  time series data values over Longyearbyen #35
+- Farmer in Malawi use case #2
+- En bruker ved miljødirektoratet vil undersøke
+  vegetasjonstilstand i et gitt område basert på målinger fra Sentinel-2 #31
+- Miljødirektoratet skal bruke dynamiske geodata
+  data i sitt beslutningssystem #16
+- A tourist guide wants to look at the cloud for a specific
+  region at night for the visibility of Northern lights #38
 
 Context diagram
-================
+===============
+
+The figure below illustrates the top level view of the architecture.
+A mock of "S-ENDA Find" is the product of Sprint 1. The system interfaces
+with data and service providers via a Web UI and a console app. The
```

Comment:
"console app" - this the "searches API", right? please make the text consistent with the figure here

Reply:
It is consistent I believe, but perhaps not clear. I'll try to describe it in a different way.